docs/phase_0.md

```diff
@@ -4,78 +4,149 @@ _Warning_: This a work in progress document with many broken links, some unfinis
+In Phase 0, a validator can be _slashed_ (a more severe penalty) if they violate the Casper FFG rules or if they create two beacon blocks in one epoch.
+
+Slashings and other events may prevent a validator from participating, but do not immediately result in the re-assignment of committees/shuffling. This is to ensure the stability of responsibilities during each epoch -- stability needs to be prioritized to ensure sufficient lookahead on duties.
+
+Slashings are like normal voluntary exits, except that their withdrawal is delayed, and penalties are applied (based on total slashings at the time). The more validators coordinate in bad behavior, the more slashings, and the more severe the penalties (see `process_slashings`).
 
 More details on slashing from a validator perspective can be found [here](https://github.com/ethereum/eth2.0-specs/blob/master/specs/validator/0_beacon-chain-validator.md#how-to-avoid-slashing).
 
 ## Eth2 parameters
 
 ### Constants
 
-[Constants](https://github.com/ethereum/eth2.0-specs/blob/v0.10.0/specs/phase0/beacon-chain.md#constants) are global parameters that are not meant to change, even between chains, but instead document numbers used in different parts of the specification.
+[Constants](https://github.com/ethereum/eth2.0-specs/blob/v0.10.0/specs/phase0/beacon-chain.md#constants) are global parameters that are not meant to change, even between chains. They document numbers used in different parts of the specification.
 
 ### Configurables
 
 [Configurable global parameters](https://github.com/ethereum/eth2.0-specs/blob/v0.10.0/specs/phase0/beacon-chain.md#configuration) are used to enable testnets to experiment, avoid collisions, and provide the necessary flexibility in development.
 
-The philosophy with configuration is that changes to values should only be between chains, not over time. Instead, additional constants can be introduced for changed functionality (e.g. if slot time were changed at Phase 1, `PHASE_1_SLOTS_PER_EPOCH` would be introduced rather than altering `SLOTS_PER_EPOCH`). Transitioning from one phase to another is not a hard reset, but a live system. Additionally, security analysis and optimizations greatly benefit from compile-time garantuees.
+The philosophy behind configuration is that changes to values should only happen between chains, not over time.
+
+This means that in order to change functionality, additional constants must be introduced. For example, if there is a decision to change the slot time in Phase 1, rather than altering `SLOTS_PER_EPOCH`, we'd have to introduce a new constant -- `PHASE_1_SLOTS_PER_EPOCH`, say, 
```

Comment:
Not 100% sure what you wanted with the trailing ", say,". Need to at least end here with a period